crates/cast/src/cmd/call.rs

```diff
@@ -148,6 +148,12 @@ pub struct CallArgs {
     /// Format: address:slot:value
     #[arg(long = "override-state-diff", value_name = "ADDRESS:SLOT:VALUE")]
     pub state_diff_overrides: Option<Vec<String>>,
+
+    /// Override the time field of a block
+    ///
+    /// Format: block:time
+    #[arg(long = "override-time", value_name = "BLOCK:TIME")]
+    pub time_overrides: Option<Vec<String>>,
```

Comment:
we can name this just `block.time` because block overrides arent't that complex, this can be a simple u64